src/frontend/posts_page/components/PostBlock.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+interface MyProps {
```

Comment:
The name here should be unique and distinguishable. May be 'PostBlockProps'.

Reply:
Got it